babel.config.js

```diff
@@ -4,17 +4,12 @@ module.exports = {
       '@babel/preset-env',
       {
         targets: {
-          node: 10,
+          node: 12,
```

Comment:
Should we go for 14 because of EOL of 12 is around now

Reply:
yes, but that's a breaking change.  kind of depends how much this will negatively impact customers

Reply:
Ok, let’s merge it then